README.md

```diff
@@ -32,7 +32,11 @@ For Typescript components, please see [ts-packages](./ts-packages).
 
 ### Developer chat
 
-You can chat to us in [Keybase](https://keybase.io). Download their chat app, then click **Teams -> Join a team**. Type **nymtech.friends** into the team name and hit **continue**. For general chat, hang out in the **#general** channel. Our development takes places in the **#dev** channel. Node operators should be in the **#node-operators** channel.
+> We used to use Keybase for developer chats, but we have since migrated to Matrix and Discord. We no longer check the old **nymtech.friends** Keybase team. 
```

Comment:
this is already present in the current master, develop, and release branches. Why is this change here?

Reply:
I don't remember to be changed this file 😕

Reply:
Do you want that I delete the file of this PR?

Reply:
yes if you can please remove those commits - it looks like this PR actually has 3 of my commits in it, at the beginning, which have already been committed to the main branches of the repo